src/module/actor/lancer-actor-sheet.ts

```diff
@@ -211,24 +211,30 @@ export class LancerActorSheet<T extends LancerActorType> extends ActorSheet<
       ev.stopPropagation();
 
       const params = ev.currentTarget.dataset;
-      const data = await this.getDataLazy();
-      if (params.path && params.writeback) {
-        const item = resolve_dotpath(data, params.path) as Counter;
-        const writeback = resolve_dotpath(data, params.writeback) as RegEntry<any>;
-        const available = params.available === "true";
-
-        if (available) {
-          // Deduct uses.
-          item.Value = item.Value > 0 ? item.Value - 1 : 0;
-        } else {
-          // Increment uses.
-          item.Value = item.Value < (item.Max || 6) ? item.Value + 1 : item.Max || 6;
-        }
+      const available = params.available === "true";
+      this._updateCounterData(params.path, params.writeback_path, available ? -1 : 1);
+    });
+  }
 
-        await writeback.writeback();
-        console.debug(item);
+  async _updateCounterData(path: string | undefined, writeback_path: string | undefined, delta: number) {
+    if (path && writeback_path) {
+      const data = await this.getDataLazy();
+      const item = resolve_dotpath(data, path) as Counter;
+      const writeback = resolve_dotpath(data, writeback_path) as RegEntry<any>;
+      const min = item.Min || 0;
+      const max = item.Max || 6;
```

Comment:
This 6 feels slightly arbitrary but I'm leaving it since it already existed there.

Reply:
I think that's fine. _Most_ counters in official content are meant to be represented by a d6, so that's a reasonable default.